specification/cosmos-db/resource-manager/Microsoft.DocumentDB/preview/2021-04-01-preview/cosmos-db.json

```diff
@@ -5202,6 +5202,270 @@
           }
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.DocumentDB/databaseAccounts/{accountName}/services/{serviceName}": {
+      "put": {
+        "tags": [
+          "DataTransfer"
+        ],
+        "x-ms-examples": {
+          "CosmosDBDataTransferClusterCreate": {
+            "$ref": "./examples/CosmosDBDataTransferClusterCreate.json"
+          }
```

Comment:
In here, we can have two examples, one for DataTransfer and second for SqlDedicatedGateway.